sklearn/preprocessing/tests/test_data.py

```diff
@@ -1966,6 +1966,24 @@ def test_normalizer_max():
         assert_almost_equal(la.norm(X_norm[3]), 0.0)
 
 
+def test_normalizer_max_sign():
+    # check that we normalize by a positive number even for negative data
```

Comment:
Can we also check for mixed data where the value with largest magnitude is negative

Reply:
Done! :+1: